cloudsim-plus/src/main/resources/vmtemplates/aws/t2.nano.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
If it doesn't require a lot of work, create a json file for every kind of Amazon EC2 instances.
There are some of them missing.